t/porting/diag.t

```diff
@@ -510,19 +510,34 @@ sub check_message {
     return $ret;
 }
 
-# Lists all missing things as of the inauguration of this script, so we
-# don't have to go from "meh" to perfect all at once.
-# 
-# PLEASE DO NOT ADD TO THIS LIST.  Instead, write an entry in
-# pod/perldiag.pod for your new (warning|error).  Nevertheless,
-# listing exceptions here when this script is not smart enough
-# to recognize the messages is not so bad, as long as there are
-# entries in perldiag.
-
-# Entries after __CATEGORIES__ are those that are in perldiag but fail the
-# severity/category test.
+# The DATA section includes two types of entries, in two sets separated by a
+# blank line.
+#
+# The first set are entries whose text we think is fully explanatory and don't
+# need further elaboration in perldiag.  This set should consist of very few
+# entries.
+#
+# The second set, after the blank line, consists of TODO entries.  (There are
+# actually two subsets described below.)  This list should basically be just
+# those entries that otherwise would have generated an error upon inauguration
+# of this script, so we didn't have to go from "meh" to perfect all at once.
```

Comment:
In comments and documentation, I generally prefer `s/script/program/` (2 lines farther on as well) -- but I recognize that here you're preserving existing language.

Reply:
Changed